todo_app/data/trello.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+import requests
+from requests.models import Response
+from todo_app.data.item import Item
+
+class TrelloUtility:
+    """ Consider making BOARD_ID dynamic"""
+    BOARD_ID = "5fb51906cc7d395b1b145b0e"
```

Comment:
This hardcoded id should be moved to the `.env` file and documented in the `README.md`

Reply:
This, too, is now done